megamek/src/megamek/common/Mounted.java

```diff
@@ -572,7 +572,7 @@ public double getTonnage(RoundWeight defaultRounding) {
         if (isDWPMounted()) {
             return defaultRounding.round(retVal * 0.75, getEntity());
         } else if (isSquadSupportWeapon()) {
-            retVal *= getEntity().isClan() ? 1.4 : 1.5;
```

Comment:
Should clan be the opposite side, as clan units are 5 member versus IS' 4?

Reply:
It's not based on squad size. It's based on tech base. Each SSW mount on a Clan suit is 40% of the total weapon weight and on an IS suit it's 50% of the weapon weight.

Reply:
See the rules TM, 270.